sorting_robot/Pose source
sorting_robot/Cell bin_position
---
sorting_robot/Pose[] path